x-pack/plugins/infra/server/lib/alerting/metric_threshold/lib/evaluate_alert.ts

```diff
@@ -26,6 +28,7 @@ interface Aggregation {
       aggregatedValue: { value: number; values?: Array<{ key: number; value: number }> };
       doc_count: number;
       to_as_string: string;
+      from_as_string: string;
       key_as_string: string;
```

Comment:
Not sure how spread this is in the codebase, but having the `as_string` suffix is redundant, as with types we already know that is a string.

Reply:
That's a type for the response returned from Elasticsearch. When you use a `date_range` aggregation it sets `from_as_string` and `to_as_string`. Before this change, we were using the `to_as_string` for the timestamp. When `date_histogram` returns this data the timestamp of the bucket would be the same as the `from_as_string`, the timestamp in a date histogram is the start of the bucket.

Reply:
Make sense then! Thanks!